generate.py

```diff
@@ -172,7 +172,7 @@ def main():
         outputs.extend(net.push_ops)
 
         print('Priming generation...')
-        for i, x in enumerate(waveform[:-(args.window + 1)]):
+        for i, x in enumerate(waveform[-args.window: -1]):
```

Comment:
Ah, I misunderstood. The old code `waveform[:-(args.window + 1)]` is not taking a portion of size args.window, it's taking everything from the beginning until args.window from the end. Which will be a different size every time, and isn't what we wanted.